interfaces/PA-App/src/app/services/padata.service.ts

```diff
@@ -167,4 +166,19 @@ export class PaDataService {
     this.setLoggedOut();
   }
 
+  public async deleteAccount(password: string): Promise<any> {
+    return new Promise((resolve, reject) => {
+      this.paAccountApi.deleteAccount(password)
```

Comment:
You can just return this line. No need to wrap it in another Promise, perse..

Apart from the logging though...